dotCMS/src/main/java/com/dotcms/languagevariable/business/LanguageVariableAPIImpl.java

```diff
@@ -0,0 +1,120 @@
+package com.dotcms.languagevariable.business;
+
+import com.dotcms.contenttype.model.type.ContentType;
+import com.dotcms.contenttype.model.type.MultilinguableFallback;
+import com.dotcms.keyvalue.business.KeyValueAPI;
+import com.dotcms.keyvalue.model.KeyValue;
+import com.dotmarketing.business.APILocator;
+import com.dotmarketing.exception.DotDataException;
+import com.dotmarketing.exception.DotSecurityException;
+import com.dotmarketing.portlets.languagesmanager.business.LanguageAPI;
+import com.dotmarketing.portlets.languagesmanager.model.Language;
+import com.dotmarketing.util.Config;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.dotmarketing.util.Config'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)